spec/lib/easybill-rest-api/api_client_spec.rb

```diff
@@ -0,0 +1,38 @@
+RSpec.describe Easybill::ApiClient do
+  subject do
+    described_class.new(config)
+  end
+
+  let(:config) do
+    c = Easybill::Configuration.new
```

Comment:
`Easybill::Configuration.new` already yields the config object, it looks a bit nicer than this.